build.gradle

```diff
@@ -125,6 +125,34 @@ publishing {
                 }
 
                 developers {
+                    developer {
+                        name = "Civitaspo"
+                        email = "civitaspo@gmail.com"
+                    }
```

Comment:
@civitaspo We're nominating you in `pom.xml`'s `<developers>` like we did in https://github.com/embulk/embulk-output-s3/pull/26#discussion_r660415407.